src/js/player.js

```diff
@@ -3267,9 +3262,12 @@ class Player extends Component {
     }
 
     let techAutoplay;
+    // manual autoplay is only supported where promises are
+    const PromiseClass = this.options_.Promise || window.Promise;
 
-    // if the value is a valid string set it to that
-    if (typeof value === 'string' && (/(any|play|muted)/).test(value)) {
+    // if the value is a valid string and we have a promise class
+    // set it to that
+    if (PromiseClass && typeof value === 'string' && (/(any|play|muted)/).test(value)) {
```

Comment:
basically, only do fancy autoplay handling if Promise is available?

Reply:
Does this mean that if you set `autoplay` to be any of these and are running on IE11, it'll stop working?

Reply:
Autoplay will just be set to a boolean in that case, which should still work on that browser anyway.

Reply:
We should think about whether IE11 is the only one that could be affected by this, because it could be a breaking change